docs/tutorials/python/index.ipynb

```diff
@@ -62,11 +64,12 @@
     "        }\n",
     "    )\n",
     "\n",
-    "def search(query):\n",
+    "def search(query, enhance_similarity=False):\n",
     "    return requests.post(\n",
     "        MEMOIRE_URL + \"/search\",\n",
     "        headers = {\"Authorization\": f\"Bearer {API_KEY}\"},\n",
     "        json = {\n",
+    "            \"enhanceSimilarity\": enhance_similarity,\n",
```

Comment:
Although, I don't think there's a need to have the variable in the function, I think it's nice to keep it in the example


ps; you should know the notebook works with the dev version of memoire too, you don't need to pull the container